README.md

```diff
@@ -2,7 +2,7 @@
 
 # kubectl-rook-ceph
 
-Provide common management and troubleshooting tools for the Rook Ceph storage provider as a Krew plugin.
+Provide common management and troubleshooting tools for the [Rook Ceph](https://github.com/rook/rook) storage provider as a [Krew](https://github.com/kubernetes-sigs/krew) plugin.
 
 ## Install
 
```

Comment:
To simplify the install instructions, how about combining lines 9-11 and no bullet point?
```
To install the plugin, run the command below:
```

Reply:
Lines 15-20 should be moved to the next section

Reply:
done.